lib/substitutions.js

```diff
@@ -69,9 +69,6 @@ var slackToMatrix = function(body, file) {
         }
     }
 
-    // remove <> from links, e.g:
-    body = body.replace(/<((https?:\/\/)?[^>]+?)>/g, '$1');
```

Comment:
oh?

Reply:
Yeah I added this because I thought its absence was causing problems, but it seems that I introduced this bug instead.